cli/support.go

```diff
@@ -225,6 +217,41 @@ func (r *RootCmd) supportBundle() *serpent.Command {
 	return cmd
 }
 
+// summarizeBundle makes a best-effort attempt to write a short summary
+// of the support bundle to the user's terminal.
+func summarizeBundle(inv *serpent.Invocation, bun *support.Bundle) {
+	if bun == nil {
+		cliui.Error(inv.Stdout, "No support bundle generated!")
+		return
+	}
+
+	if bun.Deployment.Config == nil {
+		cliui.Error(inv.Stdout, "No deployment configuration available!")
+		return
+	}
+
+	docsURL := bun.Deployment.Config.Values.DocsURL.String()
+	if bun.Deployment.HealthReport == nil {
+		cliui.Error(inv.Stdout, "No deployment health report available!")
+		return
+	}
+
+	deployHealthSummary := bun.Deployment.HealthReport.Summarize(docsURL)
```

Comment:
Nit: logical grouping

```suggestion
	if bun.Deployment.HealthReport == nil {
		cliui.Error(inv.Stdout, "No deployment health report available!")
		return
	}
	deployHealthSummary := bun.Deployment.HealthReport.Summarize(docsURL)
```